package.json

```diff
@@ -26,16 +26,26 @@
   "homepage": "https://github.com/UKHomeOffice/lev-react-components#readme",
   "devDependencies": {
     "@babel/core": "^7.4.5",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "^7.8.3",
     "@svgr/rollup": "^4.3.0",
+    "@testing-library/jest-dom": "^5.1.1",
+    "@testing-library/react": "^9.5.0",
+    "babel-jest": "^25.1.0",
     "babel-plugin-styled-components": "^1.10.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Is this being used? I didn't spot it.

Reply:
It doesn't seem to be needed - I'll bin it

Reply:
Fixed